### PR TITLE
Documentation/release test plan 17072024

### PR DESCRIPTION
**Pull request by Faizi52996**
release test plan documentation

### CODE REVIEW

File: documentation/testing/test-plans/20240717_release-test-plan.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Sanity Testing
+
+### Approach
+#### Manual functional Testing: 
+Manual functional testing will be carried out to validate fixes and updates are in place in pre-prod. 
+#### Sanity Testing
+This testing will be carried out in pre-prod to ensure all functionalities are working as expected. 
+
+### Test Scope
+#### Issues to be Tested:
```
Comment (jrabbott) [inline documentation/testing/test-plans/20240717_release-test-plan.md:22]: This doesn't list all the issues. WAF changes included.

File: documentation/testing/test-plans/20240717_release-test-plan.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Exit Criteria:
+- All tests have been completed, with any issues found logged in the product backlog
+- Issue priority agreed with Product owner;
+    - Any new critical issues found during release test are resolved prior to release and retested
+    - Any new major/minor issues found during release test will be scheduled for next release
+
+
+### Risk Management
+#### Risk Identification:
```
Comment (jrabbott) [inline documentation/testing/test-plans/20240717_release-test-plan.md:52]: Risks with WAF changes.

File: documentation/testing/test-plans/20240717_release-test-plan.md (hunk 1 above)
Comment (Faizi52996) [reply documentation/testing/test-plans/20240717_release-test-plan.md]: updated it.

File: documentation/testing/test-plans/20240717_release-test-plan.md (hunk 2 above)
Comment (Faizi52996) [reply documentation/testing/test-plans/20240717_release-test-plan.md:52]: updated it accordingly.

**Review by jrabbott — APPROVED** [review]

**Pull request merged**